regression-tests/tests/cname-to-nxdomain-any/expected_result.dnssec

```diff
@@ -1,5 +1,7 @@
 0	nxd.example.com.	120	IN	CNAME	nxdomain.example.com.
 0	nxd.example.com.	120	IN	RRSIG	CNAME 13 3 120 [expiry] [inception] [keytag] example.com. ...
+0	nxd.example.com.	86400	IN	NSEC	outpost.example.com. CNAME RRSIG NSEC
```

Comment:
this extra output is unexpected. If anything I would expect some responses to become smaller!

Reply:
i wonder if we should be setting `weDone` when we match-but-don't-follow a CNAME?